glnx-errors.h

```diff
@@ -109,7 +109,8 @@ typedef struct {
 static inline void
 glnx_cleanup_auto_prefix_error (GLnxAutoErrorPrefix *prefix)
 {
-  g_prefix_error (prefix->error, "%s: ", prefix->prefix);
+  if (prefix->error && *(prefix->error))
```

Comment:
Mmm...OK, I do random performance things too, but my basic rule is more like "avoid doing silly things inside loops".  We make *tons* of function calls into glib (and other libraries).  I haven't run a profiler on either ostree or rpm-ostree in a while (i'm suspecting you haven't either) and I'm certain doing so would find some absolutely silly stuff in places we didn't think of.

I'm fine with this patch if you prefer it, but I'd hesitate to do a lot more of this.  WDYT?